src/LookingGlassConfig.ts

```diff
@@ -18,6 +18,8 @@ import * as HoloPlayCore from "holoplay-core"
 
 export const DefaultEyeHeight: number = 1.6
 
+let quiltResolution = 3840
```

Comment:
for now, the default quilt resolution is 3840 x 3840, unless an 8K resolution Looking Glass display is detected, in which case the value will return 8192 x 8192. 

This will be improved in a future release to use the settings sent by Looking Glass Bridge, and fallback if a user is still using HoloPlay Service.